charts/verdaccio/templates/deployment.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
   {{- end }}
 spec:
   {{- if .Values.replicaCountEnabled }}
-  replicas: {{ default 1 .Values.replicaCount }}
+  replicas: {{ .Values.replicaCount }}
```

Comment:
I'd assume since on Values.yaml `replicaCount: 1` it would be the default anyway right?

Reply:
correct, but this way it can be changed